app-web/qbittorrent/02-nox/defines

```diff
@@ -2,6 +2,12 @@ PKGNAME=qbittorrent-nox
 PKGSEC=web
 PKGDEP="libtorrent-rasterbar qt-5"
 BUILDDEP="boost"
-PKGDES="Free and reliable command line P2P Bittorrent client"
+PKGDES="Free and reliable command line P2P Bittorrent client (headless version)"
```

Comment:
This should be considered a Web interface, not "headless."

Reply:
I would also recommend renaming the executable to something like `qbittorrent-console`.

Reply:
> This should be considered a Web interface, not "headless."

Please suggest a better description...?

> I would also recommend renaming the executable to something like qbittorrent-console.

I think `qbittorrent-nox` is appropriate as [qBittorrent use `qbittorrent-nox` in their wiki](https://github.com/qbittorrent/qBittorrent/wiki/Running-qBittorrent-without-X-server-(WebUI-only)#running).

Reply:
Sounds good on the executable name.

As for descriptions, please use the following:

- `qbittorrent`: A P2P (point-to-point) Bittorrent client
- `qbittorrent-nox`: A P2P (point-to-point) Bittorrent client (Web interface)